autobotAI_integrations/integrations/proofpoint_dlp/code_sample.py

```
@@ -0,0 +1,29 @@
+        query_params = {"limit": limit}
+        if created_after:
+            query_params["created_after"] = created_after
+
+        return [
+            {
+                "status": "ok",
+                "params_received": list(params.keys()),
+                "query_params": query_params,
+            }
+        ]
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)